src/storm/mesos/MesosSupervisor.java

```diff
@@ -182,6 +194,10 @@ public void disconnected(ExecutorDriver driver) {
 
   }
 
+  protected boolean startLogViewer(Map conf) {
+     return Optional.fromNullable((Boolean) conf.get(MesosCommon.AUTO_START_LOGVIEWER_CONF)).or(true);
```

Comment:
I think in the future we should make some helpers to automatically get default value